diff --git a/dist/changesets-assemble-release-plan.cjs.js b/dist/changesets-assemble-release-plan.cjs.js
index a91112a7ddf4b6f242965e0bf99ba2f60f988ede..6baddab89d4a41754c8f73d5d914604a54f248cb 100644
--- a/dist/changesets-assemble-release-plan.cjs.js
+++ b/dist/changesets-assemble-release-plan.cjs.js
@@ -336,6 +336,10 @@ function shouldBumpMajor({
   preInfo,
   onlyUpdatePeerDependentsWhenOutOfRange
 }) {
+  if (depType === "peerDependencies") {
+    return false;
+  }
+
   // we check if it is a peerDependency because if it is, our dependent bump type might need to be major.
   return depType === "peerDependencies" && nextRelease.type !== "none" && nextRelease.type !== "patch" && (
   // 1. If onlyUpdatePeerDependentsWhenOutOfRange set to true, bump major if the version is leaving the range.
